sonic_installer/main.py

```diff
@@ -637,6 +637,35 @@ def set_next_boot(image):
         sys.exit(1)
     bootloader.set_next_image(image)
 
+# Set fips for image
+@sonic_installer.command('set-fips')
+@click.argument('image')
+@click.option('--disable-fips', is_flag=True,
```

Comment:
>disable [](http://example.com/codeflow?start=17&length=7)

prefer positive words, like `--enable-fips`

Reply:
Can we set the enable-fips default value to True?